kit/charts/atk/charts/dapp/values.yaml

```
@@ -172,3 +172,55 @@ initContainer:
+  resources:
+    requests:
+      memory: "2Gi"
+      cpu: "250m"
+    limits:
+      memory: "3Gi"
+
+  workspace:
+    # Size of the workspace PVC
+    size: 1Gi
+    # Storage class to use for the workspace PVC
+    storageClass: local-path
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)